bin/vyper

```diff
@@ -76,18 +84,34 @@ elif not args.debug:
     sys.tracebacklimit = 0
 
 
-def uniq(seq):
-    exists = set()
-    return [x for x in seq if not (x in exists or exists.add(x))]
+T = TypeVar('T')
+
+
+def uniq(seq: Iterable[T]) -> List[T]:
+    """
+    Returns unique items in ``seq`` in order.
```

Comment:
*Nitpick*: We use imperative present tense among the rest of the Python libraries and I *think* this should also follow this. (Same as Python docs and many others)

https://github.com/ethereum/snake-charmers-tactical-manual/blob/master/documentation.md#grammar

>Return unique items in ``seq`` in order

As a rule of thumb, imperative present tense comes out when it fits in this:

>If we call this API it will: __________________________